src/platform/ESP32/NetworkCommissioningDriver.h

```diff
@@ -136,6 +136,35 @@ class ESPWiFiDriver final : public WiFiDriver
     uint16_t mLastDisconnectedReason;
 };
 
+class ESPEthernetDriver final : public EthernetDriver
+{
+public:
+    class EthernetNetworkIterator final : public NetworkIterator
+    {
+    public:
+        EthernetNetworkIterator(ESPEthernetDriver * aDriver) : mDriver(aDriver) {}
+        size_t Count() { return 1; }
+        bool Next(Network & item) { return true; }
+        void Release() override { delete this; }
+        ~EthernetNetworkIterator() = default;
+
+    private:
+        ESPEthernetDriver * mDriver;
+    };
+
+    // BaseDriver
+    NetworkIterator * GetNetworks() override { return new EthernetNetworkIterator(this); }
+    uint8_t GetMaxNetworks() { return 1; }
+    CHIP_ERROR Init(NetworkStatusChangeCallback * networkStatusChangeCallback) { return CHIP_NO_ERROR; }
+    void Shutdown() {}
```

Comment:
Where are these Init and Shutdown called?  I don't see them being called.

Reply:
Sure, but it might be worth documenting why they are there and do nothing and are never called.